tests/integration_tests/util.py

```diff
@@ -352,29 +355,13 @@ def _verify_clean_boot(
             f"{out.stdout}\nstderr:\n{out.stderr}"
         )
     else:
-        # we know that we should have a return code of 2
+        # Look at the commit history before making changes here.
+        # Behavior on Jenkins, GH CI, and local runs must be considered
+        # for all series.
         out = instance.execute("cloud-init status --long")
         rc = 2
-        # CI on main doesn't patch out this behavior so despite running on
-        # old releases it behaves as tip of main does
-        #
-        # GITHUB_BASE_REF is not defined if
-        #  1. The event that triggers a workflow run on GH CI is not
-        #     pull_request or pull_request_target.
-        #  2. The integration tests aren't run on GH but on Jenkins or locally
-        #     on a developer's machine.
-        #
-        # On Jenkins, rc should be 0. But, on GH CI on runs that are not
-        # not triggered from a pr event, we still want rc 2.
         if CURRENT_RELEASE < NOBLE:
-            if os.environ.get("ON_JENKINS"):
-                # We are on Jenkins testing against full packages
-                # Old releases return 0 for backwards compatibility
-                rc = 0
-            elif os.environ.get("GITHUB_ACTION"):
-                # On GH actions, integrations tests run on LXD without quilt
-                # patches
-                rc = 2
+            rc = 0
```

Comment:
Is this a reasonable assumption now?

Reply:
Now that 0f77b3c13a2f44c224c5dea18d27bbfa4898e39a landed moving us to oracular which is RC = 2 this should be reasonable and no need to check GITHUB_ACTION.